malindex.py

```diff
@@ -197,5 +197,5 @@ def fid2path(self, fid):
         if not self.paths:
             with open(self.pathlistfile, "rb") as f:
                 for l in f:
-                    self.paths.append(l.split(b" ")[0].decode().rstrip("\n"))
+                    self.paths.append(l.strip().decode())
```

Comment:
This does not respect whitespace trimming, right? Any space (also at the end of a file) will be part of the path.